### PR TITLE
FirstCommit

### PR DESCRIPTION
**Pull request by ArthurSoftvision1**
Pull request with HTML

### CODE REVIEW

File: FirstProject/new-evaluation.html [hunk 1]
```diff
@@ -4,65 +4,533 @@
...
-                    <a href="#" style="float:right;">Logout</a>
-
-                </div>
+    <nav class="row">
+        <div class="col-md-12" style="border-bottom: 2px solid darkgrey; padding-top: 12px;">
+            <div class="col-md-4" style="text-align:center;">
+                <img style="float:left; width: 12%;" href="#" style="width: 40%;" src="img/logo-v2.png"/>
+            </div>
+            <div class="col-md-4" style="padding-top: 8px;">
+                <a href="#">Evaluations</a>
```
Comment (zuldy) [inline FirstProject/new-evaluation.html]: `href` attributes for these links should lead to the other pages in the app

File: FirstProject/new-evaluation.html [hunk 2]
```diff
@@ -4,65 +4,533 @@
...
-                    <a href="#" style="margin-left: 100px;">New Evaluation</a>
-                </div>
-                <div class="col-md-4">
-                    <a href="#" style="float:right;">Logout</a>
-
-                </div>
+    <nav class="row">
+        <div class="col-md-12" style="border-bottom: 2px solid darkgrey; padding-top: 12px;">
+            <div class="col-md-4" style="text-align:center;">
+                <img style="float:left; width: 12%;" href="#" style="width: 40%;" src="img/logo-v2.png"/>
```
Comment (zuldy) [inline FirstProject/new-evaluation.html]: `img` tags have no `href` attribute

File: FirstProject/new-evaluation.html [hunk 3]
```diff
@@ -4,65 +4,533 @@
...
 Describe any guidance or management experience
 " ></textarea>
         </div>
 
+        <form class="language_form" style="text-align:center;">
+            <fieldset class="field_section" style="display:inline;">
+                <legend>OOP, Design Patterns:</legend>
+                    <ul style="display: inline-flex; text-align: right">
+                        <li>
+                            Classes <select>
```
Comment (zuldy) [inline FirstProject/new-evaluation.html]: `select` tags should have a name attribute so the form knows what and how to gather data to send to the server

File: FirstProject/new-evaluation.html [hunk 4]
```diff
@@ -4,65 +4,533 @@
...
+                            <option selected disabled>Evaluation</option>
+                            <option>0</option>
```
Comment (zuldy) [inline FirstProject/new-evaluation.html]: `option` tags should have a `value` attribute to know which value the user selected

File: FirstProject/new-evaluation.html [hunk 5]
```diff
@@ -4,65 +4,533 @@
...
+                    </li>
+                </ul>
+            </fieldset> <br>
+            <input class="submit_button" type="submit" value="SUBMIT" style="float: right; margin-top: 30px; padding: 15px 40px; margin-right: 100px;">
+        </form>
     </div>
-</body>
 
+    <footer class="container-fluid footer_side" style="margin-top: 80px;">
+        <div class="col-md-4">
```
Comment (zuldy) [inline FirstProject/new-evaluation.html]: These `div` tags are not necessary in this context. You can align the `footer` text using other, simpler methods

File: FirstProject/new-evaluation.html [hunk 6]
```diff
@@ -4,65 +4,533 @@
...
+                </div>
+                <div class="col-md-3">
+                    <h4 style="color:white;">Senior</h4>
+                </div>
+        </div>
+
+        <div class="row_section" style="border: 1px solid black;">
+            <div class="col-md-3">
+                <ul>
+                    <li class="bullets"><p>&#9711;</p></li>
```
Comment (zuldy) [inline FirstProject/new-evaluation.html]: These `bullets` should in fact be radio buttons with `name` and `value`

**Review by zuldy — CHANGES_REQUESTED** [review]

File: FirstProject/new-evaluation.html (hunk 2 above)
Comment (ArthurSoftvision1) [reply FirstProject/new-evaluation.html]: Ok, o sa pun path-urile spre celelalte pagini.

File: FirstProject/new-evaluation.html (hunk 1 above)
Comment (ArthurSoftvision1) [reply FirstProject/new-evaluation.html]: Am modificat.

File: FirstProject/new-evaluation.html (hunk 6 above)
Comment (ArthurSoftvision1) [reply FirstProject/new-evaluation.html]: 👍

File: FirstProject/new-evaluation.html (hunk 3 above)
Comment (ArthurSoftvision1) [reply FirstProject/new-evaluation.html]: 👍

File: FirstProject/new-evaluation.html (hunk 5 above)
Comment (ArthurSoftvision1) [reply FirstProject/new-evaluation.html]: 👍

File: FirstProject/new-evaluation.html (hunk 4 above)
Comment (ArthurSoftvision1) [reply FirstProject/new-evaluation.html]: 👍

**Pull request merged**